libs/model/src/main/kotlin/de/cyface/model/RequestMetaData.kt

```diff
@@ -39,6 +39,9 @@ import java.nio.charset.Charset
  * @property endLocation The `GeoLocation` at the end of the track represented by the transmitted measurement.
  * @property modality The modality type used to capture the measurement.
  * @property formatVersion The format version of the upload file.
+ * @property logCount The number of logs captured for this measurement.
```

Comment:
What is a log? Why do I need to capture this? Maybe explain the why also for image and video count.